File: dieta-app SPRING/src/main/java/com/dietaapp/modelo/Paciente.java
package com.dietaapp.modelo;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.time.LocalDate;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class Paciente {
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  int idPaciente;
  String primerNombre;
  String segundoNombre;
  String primerApellido;
  String segundoApellido;
  String correo;
  String numeroCelular;
  String genero; //Masculino, Femenino
  LocalDate fechaNacimiento;
}

File: app-dieta/src/app/clases/Paciente.ts
export class Paciente {
    idPaciente: number;
    primerNombre: string;
    segundoNombre: string;
    primerApellido: string;
    segundoApellido: string;
    correo: string;
    numeroCelular: string;
    genero: string; //Masculino, FemeninoString 
    fechaNacimiento: Date;
    constructor( primerNombre : string, primerApellido : string, 
        segundoNombre: string, segundoApellido : 
        string, correo : string, numeroCelular : string, 
        genero : string, fechaNacimiento : Date){
            this.primerNombre = primerNombre;
            this.primerApellido = primerApellido;
            this.segundoNombre = segundoNombre;
            this.segundoApellido = segundoApellido;
            this.correo = correo;
            this.numeroCelular = numeroCelular;
            this.genero = genero;
            this.fechaNacimiento = fechaNacimiento;
        }
}